website/docs/faq.mdx

```diff
@@ -297,6 +297,27 @@ updated the icons themselves. A backup of the current config can be found in the
 
 This is a known problem with Xonsh. The issue is tracked [here][xonsh-issue].
 
+### Visual Studio Code: ASCII Escape Codes after prompt
+
+After changing your profile to use Oh My Posh, in Visual Studio Code, a new terminal session may show unexpected
+ASCII escape sequences like this:
+
+```powershell
+\x1b7\x1b[38\x3b2\x3b89\x3b201\x3b165m \x1b[0m\x1b[38\x3b2\x3b224\x3b222\x3b244min \x1b[38\x3b2\x3b75\x3b149\x3b233m\x1b[1mpwsh\x1b[22m\x1b[27m\x1b[38\x3b2\x3b224\x3b222\x3b244m \x1b[0m\x1b[38\x3b2\x3b224\x3b222\x3b244mat \x1b[38\x3b2\x3b75\x3b149\x3b233m\x1b[1m19:24:06\x1b[22m\x1b[0m\x1b8
+```
+
+The solution is to add the following to the PowerShell configuration in Visual Studio Code settings.json:
+
+```json
+"args": [
```

Comment:
the actual solution is to disable shell integrations and restart vscode:

{
  "terminal.integrated.shellIntegration.enabled": false
}